products/eslint.md

```diff
@@ -0,0 +1,103 @@
+---
+title: eslint
+category: framework
+iconSlug: eslint
+permalink: /eslint
+releasePolicyLink: https://eslint.org/version-support/
+changelogTemplate: "https://github.com/eslint/eslint/releases/tag/v__LATEST__"
+releaseColumn: true
+releaseDateColumn: true
+eolColumn:  Maintenance Support
+eoasColumn: Active Support
+tags: herodevs
```

Comment:
```suggestion
```

I don't think we want to put too much emphasize on third-party support in endoflife.date.